WordPress/src/main/java/org/wordpress/android/ui/notifications/utils/FormattableContentClickHandler.kt

```diff
@@ -0,0 +1,133 @@
+package org.wordpress.android.ui.notifications.utils
+
+import android.content.Intent
+import android.support.v4.app.FragmentActivity
+import android.text.TextUtils
+import org.wordpress.android.R
+import org.wordpress.android.datasets.ReaderPostTable
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.SiteStore
+import org.wordpress.android.fluxc.tools.FormattableRange
+import org.wordpress.android.fluxc.tools.FormattableRangeType
+import org.wordpress.android.ui.ActivityLauncher
+import org.wordpress.android.ui.WPWebViewActivity
+import org.wordpress.android.ui.reader.ReaderActivityLauncher
+import org.wordpress.android.ui.reader.utils.ReaderUtils
+import org.wordpress.android.ui.stats.StatsAbstractFragment
+import org.wordpress.android.ui.stats.StatsActivity
+import org.wordpress.android.ui.stats.StatsTimeframe
+import org.wordpress.android.ui.stats.StatsViewAllActivity
+import org.wordpress.android.ui.stats.StatsViewType
+import org.wordpress.android.util.ToastUtils
+import javax.inject.Inject
+
+class FormattableContentClickHandler
+@Inject constructor(val siteStore: SiteStore) {
+    private val domainWpCom = "wordpress.com"
```

Comment:
We may want to consider using `private const val DOMAIN_WP_COM = "wordpress.com"`, since it's a compile time constant. Wdyt?

Reply:
I'd have to move it out of this class (or to a companion object) but why not 👍